docs/propagation.md

```diff
@@ -210,3 +217,39 @@ along F0, propagate `["c"]` along F1, and propagate nothing along F2.
 T0  | "a", **"b"** | **"c"**  | "f" |
 T1  | "a", "b"     | "c", "d" | "g" |
 T2  | **"a", "b"** | "c", "e" |     |
+
+### Data flow edge ops
+
+The above propagation step description applies to most ops. However, there are
+cases where a sharding rule is not appropriate. For those cases, Shardy defines
+_data flow edge_ ops.
```

Comment:
The DataFlowEgdeOp is an implementation detail,  I think you mean data flow op here, ditto in line 231.

We should also elaborate a bit on what information is specified for a data flow op via the ShardableDataFlowOpInterface, you can use the doc of that interface as reference.

Reply:
Sounds good, will do.